Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:20.04
+
+# Prepare system packages
+RUN apt-get update
```

Comment:
Forgot to mention this earlier: it's better to run apt-get update and install on a single line (e.g. `RUN apt-get update && apt-get install libgomp1 -y`) due to caching issues that can arise otherwise. See here: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run